utils/dependencies.go

```diff
@@ -65,7 +70,7 @@ func NewDependencyRecord(projectPath string,
 }
 
 func LocationIsBinding(location string) bool {
-	if strings.HasPrefix(location, "/whisk.system") || strings.HasPrefix(location, "whisk.system") {
+	if strings.HasPrefix(location, "/"+WHISK_SYSTEM) || strings.HasPrefix(location, WHISK_SYSTEM) || strings.HasPrefix(location, "/") {
```

Comment:
was not aware we allowed whisk.system without the "/"; is this something allowed in general or just wskdeploy?

Reply:
doesn't look like `whisk.system` is allowed as is in creating package binding, namespace is always identified with `/`, let me fix this.

```
wsk package bind /whisk.system/samples valhallaSamples --param place Valhalla
ok: created binding valhallaSamples
wsk action invoke --result valhallaSamples/greeting --param name Odin
{
    "payload": "Hello, Odin from Valhalla!"
}
```